routes/offerings.js

```diff
@@ -0,0 +1,133 @@
+const express = require("express");
+const router = express.Router();
+const Offering = require("../models/Offering");
+const Posting = require("../models/Posting");
+const Notifications = require("../utils/notificationsUtil");
+
+// http://localhost/api/offerings/{offeringId}/accept ///////////////////////////////////////////////////////
+// POST
+// Accepts the indicated offering
+router.post("/:offeringId/accept", async (req, res) => {
+  try {
+    const offering = await Offering.findOneAndUpdate(
+      {
+        _id: req.params.offeringId,
+      },
+      {
+        $set: { status: "ACCEPTED" },
+      },
+      {
+        new: true,
+      }
+    );
+
+    await Notifications(
+      offering.userId,
+      "offeringAccepted",
+      "Your offering has been accepted!"
```

Comment:
Can we somehow make the notification content personal and relevant? E.g. instead of "Your offering has been accepted!", we should use the actual user's userName (so use the person's username who is doing the accepting, can be passed in from the front-end easily to the request body since the accepting action occurs for the user who is currently logged in), also use the url of the posting passed in from the front-end, and make the notification message something like ``${userName} has accepted your offering to the posting ${postingUrl}``